content/webapp/services/prismic/fetch/visual-stories.ts

```diff
@@ -13,8 +12,7 @@ const fetchLinks = [
 const visualStoriesFetcher = fetcher<VisualStoryDocument>(
   'visual-stories',
   fetchLinks
-  );
+);
 
 export const fetchVisualStory = visualStoriesFetcher.getById;
 export const fetchVisualStories = visualStoriesFetcher.getByType;
-
```

Comment:
I'm sure I've mentioned this before (and probably a me problem) but I think my editor will add an empty line at the end of a file – not sure if this is a Prettier/eslint setting, or something baked into my VS Code settings

Reply:
@gestchild and I noticed we didn't have the same linting either, reckon we'll sit together next week and figure out what's going on cause this is a PITA.

Reply:
mine adds an empty line too

Reply:
Did yours flag the unused imports?

Reply:
This is in `.editorconfig` which is checked in. Not sure what determines whether your editor chooses to honour it though: https://github.com/wellcomecollection/wellcomecollection.org/blob/main/.editorconfig#L8

Reply:
the editorconfig docs suggest that you need a plugin for VSCode for it to work. Although I don't appear to have said plugin.

AHA! 
<img width="544" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/1394592/59f1625a-9e24-4b46-bd91-ef9d11f27fc4">

So it currently is happening for me because I've turned it on in VS Code. I wonder if there's a fool-proof way of making this happen without recourse to plugins?

Reply:
I feel like the fact that you need have have a plugin for .editorconfig to work in VS Code muddies the waters and we'd maybe be better of without it to avoid inconsistencies?